planner/core/rule_aggregation_elimination.go

```diff
@@ -36,6 +36,19 @@ type aggregationEliminateChecker struct {
 // For count(expr), sum(expr), avg(expr), count(distinct expr, [expr...]) we may need to rewrite the expr. Details are shown below.
 // If we can eliminate agg successful, we return a projection. Else we return a nil pointer.
 func (a *aggregationEliminateChecker) tryToEliminateAggregation(agg *LogicalAggregation) *LogicalProjection {
+	for _, af := range agg.AggFuncs {
+		// TODO: Actually, we can rewrite GROUP_CONCAT.
+		// When it accepts only 1 argument, we can extract this argument into a
```

Comment:
This case is easy to be added in this pr?

Reply:
Noop, since group_concat_max_len is configurable.
We can not ensure whether the length of the final result exceeds max_len even if there is only 1 argument.

And, maybe it'll be easier to implement if we wrap the argument(s) into a concat_ws despite the count of the arguments.